src/Core/assets/stuntman.css

```diff
@@ -182,16 +183,18 @@
 .stuntman-widget .stuntman-body ul li.stuntman-active .stuntman-item h3:after {
   color: limegreen;
   content: "\25CF";
-  font-size: 36px;
   position: absolute;
-  top: -12px;
-  right: 15px;
+  font-size: 200%;
+  line-height: .5em;
+  top: 50%;
+  transform: translateY(-50%);
+  right: -23;
```

Comment:
Would you add **px** to this and the other one in this PR? I had some issues before with this and added the **px** text.